msamp/operators/dist_op/dist.cpp

```diff
@@ -206,8 +205,8 @@ PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
                 return id;
             }));
     py::class_<ncclCommPtr>(m, "ncclCommPtr").def(py::init<>());
-    m.def("reduce", &reduce, "Reduce");
-    m.def("all_reduce", &all_reduce, "All reduce");
+    m.def("reduce", &dist_reduce, "Reduce");
+    m.def("all_reduce", &dist_all_reduce, "All reduce");
```

Comment:
maybe rename to custom_reduce and custom_all_reduce?

Reply:
Done. I have renamed them.